### PR TITLE
[BUGFIX] Updated interface

### PR DESCRIPTION
**Pull request by George-Farrell**
Updated RegistryInterface to ManagerRegistry

### CODE REVIEW

File: src/Repository/Authentication/UserRepository.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
  */
 class UserRepository extends ServiceEntityRepository
 {
-    public function __construct(RegistryInterface $registry)
+    public function __construct(ManagerRegistry $registry)
```
Comment (JKetelaar) [inline src/Repository/Authentication/UserRepository.php:17]: I'd rather keep this to an interface, or is this change required?

**Comment by George-Farrell** [comment]
Required, the interface is replaced with this manager I believe.

**Review by JKetelaar — APPROVED** [review]
